src/main/bash/sdkman-env.sh

```diff
@@ -17,27 +17,67 @@
 #
 
 function __sdk_env() {
-	readonly sdkmanrc='.sdkmanrc'
+	SDKMANRC=".sdkmanrc"
 
-	if [[ ! -f "$sdkmanrc" ]]; then
-		__sdkman_echo_red "No $sdkmanrc file found."
+	if [[ "$1" == "init" ]]; then
```

Comment:
What exactly is `"$1"` here? I would suggest assigning whatever it is to a local variable and then using that here instead.

Reply:
Assigned $1 to a variable.